product/linux_trusty.cfg

```diff
@@ -0,0 +1,42 @@
+#For BXT-P on clear linux
+EVMM_CMPL_FLAGS :=
+#EVMM_CMPL_FLAGS += -DDEBUG
+
+EVMM_CMPL_FLAGS += -DLOG_LEVEL=3
```

Comment:
are you sure log level 3 is required? normally we set it 0 for release.

Reply:
change to 0, updated